package.json

```diff
@@ -34,7 +40,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged --write src",
+      "pre-commit": "npx pretty-quick --staged --write src",
```

Comment:
Doesn't seem like this should be required. They don't use npx in the [docs](https://www.npmjs.com/package/pretty-quick#pre-commit-hook)

Reply:
So here is where I got this tip from
https://stackoverflow.com/questions/61196309/husky-4-x-not-working-with-visual-studio-git

Although not for VS the issue described is the same issue I was having. That being it can't find lint because its looking for it in C: 

Unsure what to do about this. Is anyone else on windows apart from me?

Reply:
It doesn't hurt anything I was just curious.

Reply:
I can also 100% keep this local if its only an issue for me and look further into what is going wrong. Really only need this if someone else has an issue